.github/ISSUE_TEMPLATE/new-defect.yaml

```diff
@@ -0,0 +1,46 @@
+name: New Defect
+description: Use this template to report a new defect.
+labels:
+  - type/defect
+  - triage
```

Comment:
The triage status and project are added automatically too all issues that aren't labelled content

Reply:
But we would add the `type/defect` label to mark this particular issue as a defect and then this automation wouldn't trigger.

Or is the workflow, that we add the `type/defect` label during triaging?

Reply:
Triage is a project column and not a label. I don't think there is a triage label at all.

Reply:
Nah, I meant getting rid of the labels wouldn't mark the issue as defect. This has to be done afterwards during "triage". But we could already add the label `type/defect`, but this wouldn't trigger the automation, because the automation only moves issued without labels to the triage column, right?

Reply:
We should add type/defect but not triage. 

The automation adds all issues to our project if the content/rawkode-live doesn't exist.

Reply:
💡 `that aren't labelled content` makes sense now. My bad :D